src/cloudant/_client_session.py

```diff
@@ -174,7 +187,12 @@ def request(self, method, url, **kwargs):
             resp.status_code == 401
         ))
 
-        if is_expired:
+        can_be_user_db_expired = (
```

Comment:
Why not add this code to the `any` block above for `is_expired`?

Reply:
I was thinking on doing that at first, but then my intention was to make it more explicit.

`is_expired` would be `True` for situations in which it is certain that the session is expired. While `can_be_user_db_expired` would not be certain that the session is expired, just that it "can be expired". Because CouchDB would throw a 404 error if the user doesn't exists or if the user that does the request is not the same user himself (like an anonymous user, or an expired session) or a server admin.

If you prefer, I can update and simplify the code to have everything in the `any` block.